src/main/kotlin/no/nav/syfo/avvistsykmelding/AvvistSykmeldingService.kt

```diff
@@ -0,0 +1,91 @@
+package no.nav.syfo.avvistsykmelding
+
+import com.fasterxml.jackson.module.kotlin.readValue
+import kotlinx.coroutines.delay
+import no.nav.syfo.ApplicationState
+import no.nav.syfo.metrics.AVVIST_SM_MOTTATT
+import no.nav.syfo.metrics.AVVIST_SM_VARSEL_OPPRETTET
+import no.nav.syfo.model.ReceivedSykmelding
+import no.nav.syfo.objectMapper
+import org.apache.kafka.clients.consumer.KafkaConsumer
+import org.apache.kafka.clients.producer.KafkaProducer
+import org.apache.kafka.clients.producer.ProducerRecord
+import org.slf4j.LoggerFactory
+import java.time.Duration
+import java.time.LocalDate.now
+import java.time.LocalDateTime
+import java.util.Collections.singletonMap
+
+private val log: org.slf4j.Logger = LoggerFactory.getLogger("no.nav.syfo.syfosmvarsel")
+
+const val OPPGAVETYPE = "0005"
+
+suspend fun opprettVarselForAvvisteSykmeldinger(
+        applicationState: ApplicationState,
+        kafkaconsumer: KafkaConsumer<String, String>,
+        kafkaproducer: KafkaProducer<String, OppgaveVarsel>,
+        oppgavevarselTopic: String,
+        tjenesterUrl: String
+) {
+    while (applicationState.running) {
+        kafkaconsumer.poll(Duration.ofMillis(0)).forEach {
```

Comment:
Hvet du hvor mange elementer den henter? 🤔

Reply:
Jeg bruker denne: https://github.com/navikt/syfosm-common/tree/master/syfosm-common-kafka

Den commiter/acker for meg, og legger evt tilbake meldingen hvis det feiler. Såvidt jeg skjønte tar den en og en av gangen. 

Lytteren stoppes ikke ved feil sånn som vi er vant til, men jeg har metrikker på antall avviste sykmeldinger som er mottatt og hvor mange varsler vi har laget, og kommer til å lage en alert som fanger opp om disse blir ulike (hvis vi leser flere enn vi oppretter varsler for). Sånn sett får vi også gratis retry 😛